### PR TITLE
Set up CI

### LINKED ISSUE #3: Set up a linter
Set up a linter to enforce code style rules.
Airbnb's rules are fine.
This issue may be resolved with #1

### LINKED ISSUE #4: Set up the CI
We need an automated pipeline for testing purposes.

### PR DESCRIPTION
**Pull request by AnastasiaPromtep**
I also added a linter with minimal configuration

Closes #4 
Closes #3

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - run: npm ci
+
+      - run: npm run lint
+        env:
+          CI: true
```
Comment (maximelouet) [inline .github/workflows/test.yaml:26]: already set by GitHub Actions

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      "Hyperlink",
+      {name: "Link", linkAttribute: "to"}
+    ]
+  },
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+    React: 'writable',
+    process: true
+  },
```
Comment (maximelouet) [inline .eslintrc.js:39]: Test without all of this

File: .github/workflows/test.yaml (hunk 1 above)
Comment (AnastasiaPromtep) [reply .github/workflows/test.yaml]: It has been removed.

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  extends: [
+    'eslint:recommended',
+    "plugin:react/recommended"
+  ],
+  settings: {
+    react: {
+      pragma: "React",
+      version: "detect"
+    }
+  },
```
Comment (maximelouet) [inline .eslintrc.js:16]: Did you test without this?

File: .eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  extends: [
+    'eslint:recommended',
+    "plugin:react/recommended"
+  ],
+  settings: {
+    react: {
+      createClass: "createReactClass", // Regex for Component Factory to use,
+                                         // default to "createReactClass"
+      pragma: "React",  // Pragma to use, default to "React"
+      version: "detect", // React version. "detect" automatically picks the version you have installed.
```
Comment (AnastasiaPromtep) [inline .eslintrc.js]: This seems to be mandatory to set the React version.
cf: https://github.com/yannickcr/eslint-plugin-react#configuration

File: .eslintrc.js (hunk 3 above)
Comment (AnastasiaPromtep) [reply .eslintrc.js:16]: I did. It give a warning about the version of React not specified

File: pages/_app.js [hunk 5]
```diff
@@ -24,4 +25,9 @@ function App({ Component, pageProps }) {
   );
 }
 
+App.propTypes = {
+  Component: PropTypes.any.isRequired,
+  pageProps: PropTypes.any.isRequired
+};
+
```
Comment (maximelouet) [inline pages/_app.js:32]: Not sure if it's relevant as the `App` component is only used by NextJS

**Pull request merged**